File: AppMapper/Profiles/StatusProfile.cs
using App.Contracts.Models.Output;
using AutoMapper;
using Domain.Contracts.Models;

namespace AppMapper.Profiles
{
    internal class StatusProfile : Profile
    {
        #region constructor

        public StatusProfile()
        {
            CreateMap<Status, StatusOutputModel>()
                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id))
                .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.Name));
        }

        #endregion
    }
}

File: UI/src/app/core/mappers/profiles/status.profile.ts
import { MappingProfile, createMap, forMember, mapFrom } from "@automapper/core";
import { StatusInputModel } from "../../models/input/status-input.model";
import { StatusModel } from "../../models/inner/status.model";

export const statusProfile: MappingProfile = (mapper) => {
    createMap(
        mapper,
        StatusInputModel,
        StatusModel,
        forMember(
            (dest) => dest.id,
            mapFrom((source) => source.id)
        ),
        forMember(
            (dest) => dest.name,
            mapFrom((source) => source.name)
        )
    );
};